cetmix_tower_server/wizards/cx_tower_plan_execute_wizard.py

```diff
@@ -27,13 +27,24 @@ class CxTowerPlanExecuteWizard(models.TransientModel):
         column2="tag_id",
         string="Tags",
     )
-    any_server = fields.Boolean()
+    any_server = fields.Boolean(default=True)
     # Lines
     plan_line_ids = fields.One2many(
         string="Commands",
         comodel_name="cx.tower.plan.line",
         compute="_compute_plan_line_ids",
     )
+    is_show_servers = fields.Boolean()
```

Comment:
Same here. Would be better not to store this field if we don't need it. 
So computed non stored field would be a better option.